### PR TITLE
Convert to TS

### PR DESCRIPTION
**Pull request by ligutanjma**

### CODE REVIEW

File: src/app/redux/Item/Item.reducers.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ const initialState = {
     items: [],
 }
 
-function items(state = initialState, action = null) {
+function items(state = initialState, action: { type: string; payload: any }) {
```
Comment (iRaySpace) [inline src/app/redux/Item/Item.reducers.ts:8]: `items` reducer should have explicit return type.
Check: https://redux.js.org/recipes/usage-with-typescript#type-checking-reducers

`action` should have type for separate interface/type
https://redux.js.org/recipes/usage-with-typescript#type-checking-actions--action-creators

File: src/app/pages/item/ItemList.tsx [hunk 2]
```diff
@@ -1,36 +1,27 @@
...
+                {items.map((item: Item) => (
                     <li key={item.id}>{item.name}</li>
                 ))}
             </ul>
         </div>
     )
 }
 
-const mapStateToProps = (state) => ({
+const mapStateToProps = (state: any) => ({
```
Comment (iRaySpace) [inline src/app/pages/item/ItemList.tsx:20]: Should have `RootState` interface/type (a union of all reducers type).
https://redux.js.org/recipes/usage-with-typescript#typing-the-connect-higher-order-component

File: src/app/pages/item/ItemList.tsx [hunk 3]
```diff
@@ -1,36 +1,27 @@
...
     items: state.items.items,
 })
 
-const mapDispatchToProps = (dispatch) => ({
+const mapDispatchToProps = (dispatch: any) => ({
```
Comment (iRaySpace) [inline src/app/pages/item/ItemList.tsx:24]: You can use types
Check: https://stackoverflow.com/questions/54844839/typescript-how-to-type-the-dispatch-in-redux

Alternatively, you can use useDispatch() of React Hocks and avoid returning whole function with dispatch
https://levelup.gitconnected.com/react-redux-hooks-useselector-and-usedispatch-f7d8c7f75cdd

File: src/app/redux/Item/Item.reducers.ts (hunk 1 above)
Comment (ligutanjma) [reply src/app/redux/Item/Item.reducers.ts:8]: Will check on this, sir @iRaySpace

**Pull request merged**